app/uk/gov/hmrc/softdrinksindustrylevy/models/FinancialLineItem.scala

```diff
@@ -17,54 +17,31 @@
 package sdil.models
 
 import java.time.{LocalDate => Date}
-import play.api.i18n.Messages
 import play.api.libs.json._
 
 sealed trait FinancialLineItem {
   def date: Date
   def amount: BigDecimal
-  def description(implicit messages: Messages): String = getClass.toString
 }
 
 case class ReturnCharge(period: ReturnPeriod, amount: BigDecimal) extends FinancialLineItem {
 
   val formatter = java.time.format.DateTimeFormatter.ofPattern("MMMM")
-
-  override def description(implicit messages: Messages): String =
-    Messages(
-      "financiallineitem.returncharge",
-      formatter.format(period.start),
-      period.end.format(formatter)
-    )
   def date = period.deadline
 }
 
-case class ReturnChargeInterest(date: Date, amount: BigDecimal) extends FinancialLineItem {
-  override def description(implicit messages: Messages): String =
-    Messages("financiallineitem.returnchargeinterest")
-}
+case class ReturnChargeInterest(date: Date, amount: BigDecimal) extends FinancialLineItem {}
 
-case class CentralAssessment(date: Date, amount: BigDecimal) extends FinancialLineItem {
-  override def description(implicit messages: Messages): String =
-    Messages("financiallineitem.centralassessment")
```

Comment:
removing these overridden defs might create and issue unless if these defs are not used